salt/modules/mac_keychain.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+'''
+Install certificates into the keychain on Mac OS
+
+'''
+from __future__ import absolute_import
+
+# Import python libs
+import logging
+
+# Import salt libs
+import salt.utils
+
+log = logging.getLogger(__name__)
+__virtualname__ = 'keychain'
+
+
+def __virtual__():
+    '''
+    Only work on Mac OS
+    '''
+    if salt.utils.is_darwin():
+        return __virtualname__
+    return False
+
+
+def install(cert, password, keychain="/Library/Keychains/System.keychain", allow_any=False, keychain_password=None):
```

Comment:
Will this password be passed in plaintext or as a hash? If it is plaintext we need to be responsible and add a note to the doc that the password can show up in the job return data